app/src/main/java/com/termux/x11/LorieService.java

```diff
@@ -212,7 +212,7 @@ public int onStartCommand(Intent intent, int flags, int startId) {
             Log.e("LorieService", action);
             terminate();
             sleep(500);
-            act.finish();
+            act.finishAffinity();
```

Comment:
`finishAffinity` is not needed. `System.exit(0);` finishes the whole process.

Reply:
Fixed the pr!